lib/screens/error_popup_dialog.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class SimpleErrorPopupDialog extends StatelessWidget {
```

Comment:
Maybe just me: By a (unspoken) convention a file name corresponds to the name of the main class inside the file. Although not required, I prefer it to be consistent.